Fix the Font-size for the accessibility concerns

najmiehsa opened the pull request:
Fixes #20361

Commit:
Fix the Font-size for the accessibility concerns

Josh-Cena commented:
@najmiehsa If your PR fixes a issue, please use "Fixes #20361" in your description

estelle committed:
Remove the entire example

10px is just way to small and is no longer recommended. I recommend getting rid of the whole section.

najmiehsa commented:
@estelle Thank you for the review. I can replace the example with a better one, 
if I know the standard that I need to follow in order to do that.

estelle committed:
Merge branch 'main' into WebAudio

najmiehsa commented:
Thank you for merging this PR ^^
